docs/penguins.nim

```diff
@@ -52,34 +54,34 @@ nbImage(url="images/penguins_map.png",
 nbText: """We do expect weight being correlated to some of the length measures
 (e.g. flipper length) with males being bigger than females.
 
-To plot this we need to remove all `NA`s:
+To plot this we need to remove all `NA` and then classify the points both by the penguins sex as
+well as their species:
 """
 # manage runtime error here!
 #nbCode:
-#  ggplot(df, aes(x="body_mass_g", y="flipper_length_mm", color = "sex", shape="species")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_sex_and_species.png")
-#nbImage(url="images/penguins_mass_vs_length_with_sex_and_species.png", caption="Penguins' mass vs flipper length")
+#  ggplot(df, aes(x="body_mass_g", y="flipper_length_mm", color = "sex", shape="species")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_sex_and_species2.png")
+#nbImage(url="images/penguins_mass_vs_length_with_sex_and_species2.png", caption="Penguins' mass vs flipper length")
```

Comment:
we probably can remove this whole commented block now that shape works :)

Reply:
Ah, yes. I wasn't sure about that, because that code still tries to plot with the NAs in the DF. And that still doesn't work (I don't actually know why, need to check what's going on). Will remove it and track it in a ggplotnim issue.